admin_interface/templatetags/admin_interface_tags.py

```diff
@@ -188,6 +188,23 @@ def admin_interface_use_changeform_tabs(adminform, inline_forms):
     return has_tabs
 
 
+@register.simple_tag(takes_context=True)
+def admin_interface_resolve_variable(context, var_name, default=""):
+    bits = var_name.split(".")
+    current = context.flatten()  # merge all and create a context dict
+
+    for bit in bits:
+        try:
+            if isinstance(current, dict):
+                current = current[bit]
+            else:
+                current = getattr(current, bit)
+        except (KeyError, AttributeError, TypeError):
+            return default
```

Comment:
`template.Variable(var_name).resolve(context)` already do all the resolution with dot syntax, so the code before is not necessary.